_admin_guides/rosco.md

```diff
@@ -165,26 +165,45 @@ aws:
         # If not specified, the default templateFile will be used.
         templateFile: aws-ebs.json
       virtualizationSettings:
-        - region: us-east-1
-          virtualizationType: hvm
-          instanceType: t2.micro
-          sourceAmi: ami-d4aed0bc
-          sshUserName: ubuntu
+      - region: us-east-1
+        virtualizationType: hvm
+        instanceType: t2.micro
+        sourceAmi: ami-d4aed0bc
+        sshUserName: ubuntu
+        spotPrice: 0
+        spotPriceAutoProduct: Linux/UNIX (Amazon VPC)
+      - region: us-west-1
+        virtualizationType: hvm
+        instanceType: t2.micro
+        sourceAmi: ami-4f285a2f
+        sshUserName: ubuntu
+        spotPrice: 0
+        spotPriceAutoProduct: Linux/UNIX (Amazon VPC)
     - baseImage:
         id: trusty
         shortDescription: v14.04
         detailedDescription: Ubuntu Trusty Tahr v14.04
         packageType: deb
+        # The following AMI ID's were retrieved from here:
+        # https://cloud-images.ubuntu.com/locator/ec2/
       virtualizationSettings:
-        - region: us-east-1
-          virtualizationType: hvm
-          instanceType: t2.micro
-          sourceAmi: ami-9eaa1cf6
-          sshUserName: ubuntu
+      - region: us-east-1
+        virtualizationType: hvm
+        instanceType: t2.micro
+        sourceAmi: ami-9d751ee7
```

Comment:
it looks like you updated the sourceAmi, is that right?

Reply:
Yes.  I did